src/main/start-main-application/lens-window/application-window/application-window.injectable.ts

```diff
@@ -22,35 +22,27 @@ const applicationWindowInjectable = getInjectable({
     const applicationName = di.inject(appNameInjectable);
     const appEventBus = di.inject(appEventBusInjectable);
     const ipcMain = di.inject(ipcMainInjectable);
-
-    const lensProxyPort = di.inject(
-      lensProxyPortInjectable,
-    );
-
-    const getContentUrl = () => `http://localhost:${lensProxyPort.get()}`;
+    const lensProxyPort = di.inject(lensProxyPortInjectable);
 
     return createLensWindow({
       id: "only-application-window",
       title: applicationName,
       defaultHeight: 900,
       defaultWidth: 1440,
-      getContentUrl,
+      getContentUrl: () => `http://localhost:${lensProxyPort.get()}`,
       resizable: true,
       windowFrameUtilitiesAreShown: isMac,
+      titleBarStyle: isMac ? "hiddenInset" : "hidden",
```

Comment:
Where is the fix exactly? Here?

Reply:
Yes, the bug was that all windows on mac were having this field forced. Which was not the case before the reference PR